tsktsk/__main__.py

```diff
@@ -13,8 +15,15 @@ def cli():
 
 
 def bot():
+    import tsktsk.commands.bot  # noqa
+
     logging.basicConfig(level=logging.INFO)
 
+    backend = get_backend("sqlite:///.tsktsk.db")
```

Comment:
Does this create a db even on cli usage?

Reply:
No its only created when its ran as a bot.

Reply:
Oh, this is in the bot main method. I thought it was the root click method for some reason 🤦‍♀️